src/js/components/projects/listItem.tsx

```diff
@@ -6,50 +6,51 @@ import { Link } from 'react-router-dom';
 
 import { ExternalLink } from '@/components/utils/';
 import { Project } from '@/store/projects/reducer';
-import { Repository } from '@/store/repositories/reducer';
 import routes from '@/utils/routes';
 
 interface Props {
-  repository: Repository;
+  repositorySlug: string;
   project: Project;
 }
 
-const ProjectListItem = ({ repository, project }: Props) => {
+const ProjectListItem = ({ repositorySlug, project }: Props) => {
   const { name, description_rendered, slug, branch_url, branch_name } = project;
 
   return (
-    <>
-      <li className="slds-item slds-p-horizontal_none slds-p-vertical_medium">
-        <h3 className="slds-text-heading_small">
-          <Link to={routes.project_detail(repository.slug, slug)}>{name}</Link>
-        </h3>
-        {description_rendered && (
-          <p
-            className="markdown"
-            dangerouslySetInnerHTML={{ __html: description_rendered }}
-          />
-        )}
-        {branch_url && (
-          <p
-            className="slds-text-body_small
+    <div className="cell-stacked slds-truncate">
+      <Link
+        to={routes.project_detail(repositorySlug, slug)}
+        className="slds-text-heading_small slds-p-bottom_xx-small project-name-link"
+      >
+        {name}
+      </Link>
+      {description_rendered && (
+        <div
+          className="markdown slds-truncate truncate-inner-items"
+          dangerouslySetInnerHTML={{ __html: description_rendered }}
+        />
+      )}
+      {branch_url && (
+        <p
```

Comment:
Are these changes that I made to the markup OK @jgerigmeyer @duggiemitchell? I think there is a <ul> wrapper somewhere we will want to remove since this is no longer a list (I converted all <li> to div and tried to unnest some of this. I couldn't figure out how to remove the first wrapper div in the table cell (if we do that now, the styles will likely need to be updated since some of them rely on nesting).

![Screen Shot 2020-05-28 at 12 51 11 AM](https://user-images.githubusercontent.com/1581694/83104229-e9ace600-a07d-11ea-8c83-515a0173f963.jpg)

Reply:
@stacyk We can't remove the `th > div`. We could remove the inner `div.cell-stacked.slds-truncate`, and move those classes up to the `th` if we want. Or we can leave it as-is.

Reply:
would we be able to add the same `cell-stacked slds-truncate` classes to the outer div? Id rather un-nest as much as we can. If not, then no need to change anything

Reply:
@stacyk No, we really have _no_ control over the outer div. We only control the `th` and the inner `div`. Like I said, we could remove the inner `div` entirely and move those classes up to the `th`, but we can't move them to the outer `div`.